packages/react-vanilla-components/src/components/Image.tsx

```diff
@@ -0,0 +1,35 @@
+// *******************************************************************************
+//  * Copyright 2024 Adobe
+//  *
+//  * Licensed under the Apache License, Version 2.0 (the “License”);
+//  * you may not use this file except in compliance with the License.
+//  * You may obtain a copy of the License at
+//  *
+//  *     http://www.apache.org/licenses/LICENSE-2.0
+//  *
+//  * Unless required by applicable law or agreed to in writing, software
+//  * distributed under the License is distributed on an “AS IS” BASIS,
+//  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  * See the License for the specific language governing permissions and
+//  * limitations under the License.
+
+//  * The BEM markup is as per the AEM core form components guidelines.
+//  * LINK- https://github.com/adobe/aem-core-forms-components/blob/master/ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/emailinput/v1/emailinput/emailinput.html
+//  ******************************************************************************
+import React from 'react';
+import { PROPS } from '../utils/type';
+import { withRuleEngine } from '../utils/withRuleEngine';
+
+const Image = (props: PROPS) => {
```

Comment:
Can you check if the image component in core component has the URL for both the scenarios (ie) dam image and inline image ? I guess in case of in line image, the src path does not come up in the image component ?

Reply:
Dam image and the image which we browse from our system both have the url(relative) in image component.